documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -140,7 +140,7 @@ As per the standard authorization code flow, the device application is redirecte
 
 The API exposure platform receives the request from the device application (Step 3) and does the following:
 
-- Use network based authentication mechanism to obtain the user identifier, i.e.: MSISDN. Set the OAuth sub to the unique user ID in the operator (Step 4).
+- Use network based authentication mechanism to obtain the subscription identifier, i.e.: MSISDN or IMSI. Set the id_token sub to some unique user ID and associate the sub with the access token. The id_token sub SHOULD not reveal information to the API consumer that they not already know, e.g. using the MSISDN as a sub might violate privacy. (Step 4).
```

Comment:
```suggestion
- Use network based authentication mechanism to obtain the user identifier,e.g.: phone number or IMSI. Set the id_token sub to some unique user ID and associate the sub with the access token. The id_token sub SHOULD not reveal information to the API consumer that they not already know, e.g. using the MSISDN as a sub might violate privacy. (Step 4).
```

Reply:
That "user" is based on the definition in the glossary. I think we should write the correct term here which is subscriber.

Reply:
Well, I think the text should be consistent with the glossary terms. Changing the glossary terms is another discussion.

Reply:
I think using the term subscription is the correct one and does not require a glossary change.
The network authentication never authenticates the end-user and using "user" here is confusing.

Reply:
Ok, updated suggestion. I keep "subscription" in this sentence.